docs/content/concepts/authorization.md

```diff
@@ -208,9 +208,59 @@ and work just like in a regular Kubernetes cluster.
 
 It is possible to bind to roles and cluster roles in the bootstrap policy from a local policy `RoleBinding` or `ClusterRoleBinding`.
 
+### Scopes
+
+Scopes are a way to limit the access of a user to a specific logical cluster. Scopes are attached to the user identity
+by adding `cluster:<logical-cluster>` to the `authentication.kcp.io/scope` extra field. The scope is then checked by
+the authorizers. For example:
+
+```yaml
+user: user1
+groups: ["group1"]
+extra:
+  authentication.kcp.io/scope: cluster:logical-cluster-1
```

Comment:
Should this be a `scopes` as it can contain multiple?

Reply:
yes, extra is `map[string][]string`. But the keys are traditionally singular.

Reply:
fixed